crates/astria-sequencer/src/proposal/commitment.rs

```diff
@@ -22,7 +22,7 @@ use proto::native::sequencer::v1alpha1::SignedTransaction;
 /// This is somewhat arbitrary, but could be useful for proof of an action within the action tree.
 pub(crate) fn generate_sequence_actions_commitment(
     txs_bytes: Vec<Bytes>,
-) -> ([u8; 32], Vec<Bytes>) {
+) -> ([u8; 32], [u8; 32], Vec<Bytes>) {
```

Comment:
It feels like this may have outgrown a tuple. It's unclear what the function of each set of u8 and bytes is here.

Can deduce from usages, but I think a struct return might be better for understanding from the function signature itself standpoint.